src/core/internal/convert.d

```diff
@@ -633,3 +633,41 @@ const(ubyte)[] toUbyte(T)(ref T val) if (is(T == struct) || is(T == union))
         return (cast(const(ubyte)*)&val)[0 .. T.sizeof];
     }
 }
+
+char[] itoa(ulong n) pure nothrow @safe
```

Comment:
It is only used in CTFE, so I didn't see the GC-centric interface as an issue.

If core.demangle and object were to use the same implementation, would core.internal.convert still be a good place for it?